hack/quickstart-setup.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+#
+# Copyright 2021 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+if [ -z $KUADRANT_BRANCH ]; then
+  KUADRANT_BRANCH=${KUADRANT_BRANCH:="main"}
+fi
+if [ -z $KUADRANT_ORG ]; then
+  KUADRANT_ORG=${KUADRANT_ORG:="kuadrant"}
+fi
+if [ -z $MGC_BRANCH ]; then
+  MGC_BRANCH=${MGC_BRANCH:="main"}
+fi
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/adam-cattermole/multicluster-gateway-controller/update-quickstart/hack/.quickstartEnv)"
+source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/adam-cattermole/multicluster-gateway-controller/update-quickstart/hack/.deployUtils)"
```

Comment:
Needs updating once this is merged https://github.com/Kuadrant/multicluster-gateway-controller/pull/731

Reply:
I've updated this now, so for the time being can be run with:

```bash
export KUADRANT_ORG=adam-cattermole
export MGC_REF=update-quickstart
curl "https://raw.githubusercontent.com/adam-cattermole/kuadrant-operator/quickstart-script/hack/quickstart-setup.sh" | bash
```